specification/marketplacecatalog/data-plane/Products/preview/2023-01-01-preview/products.json

```diff
@@ -0,0 +1,1818 @@
+{
+  "x-generator": "NSwag v13.12.1.0 (NJsonSchema v10.4.6.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Catalog Browse API",
+    "description": "Microsoft Marketplace Catalog Browse API",
+    "termsOfService": "None",
+    "version": "2023-01-01-preview"
+  },
+  "host": "catalogapi.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/products/{id}": {
+      "get": {
+        "tags": [
+          "SinglePublicProduct"
+        ],
+        "summary": "Single Public Product API provides an unauthenticated endpoint for consuming product attributes of a single commercial Marketplace public product. Each API request must include an API key allocated to you. Send an email to MKPL_Platform_API_DL@microsoft.com including the following details to get the API Key:\n- Customer name\n- Service/Product name\n- AD Registered App ID (if available)\n- Focal point email\n- Use case/scenario\n- Expected traffic volume, including peak requests per second (daily)",
+        "operationId": "SinglePublicProduct_Get",
+        "x-ms-examples": {
+          "Product_Get": {
+            "$ref": "./examples/SinglePublicProduct_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "type": "string",
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "description": "The unique product Id"
+          },
+          {
+            "type": "string",
+            "name": "market",
+            "in": "query",
+            "required": true,
+            "description": "Product market value  (the response will include only products that can be sold in the specified market)\nPossible values can be found at https://docs.microsoft.com/en-us/azure/marketplace/marketplace-geo-availability-currencies. Example: 'US'"
+          },
+          {
+            "type": "string",
+            "name": "api-version",
+            "x-originalName": "apiVersion",
+            "in": "query",
+            "required": true,
+            "description": "Version of the REST API used for the request. "
+          },
+          {
+            "type": "string",
+            "name": "language",
+            "in": "query",
+            "description": "Language to search, ISO 639-1 two-letter code, possible values - 'en,cs,de,es,fr,hu,it,ja,ko,nl,pl,pt-br,pt-pt,ru,sv,tr,zh-hans,zh-hant'. Default is \"en\"",
+            "default": "en"
+          },
+          {
+            "type": "boolean",
+            "name": "includeStopSoldPlans",
+            "in": "query",
+            "description": "Denotes whether to include stop sold plans",
+            "default": false
+          },
+          {
+            "type": "array",
+            "name": "hideKeys",
+            "in": "query",
+            "description": "Add hide key to retrieve preview items. Hidekeys are provided in 'multi' format: &Hidekeys[]=\"first\"&Hidekeys[]=\"second\"",
```

Comment:
I am not really sure the example in description is correct...

Also see Mark's comment https://github.com/Azure/azure-rest-api-specs/issues/21533#issuecomment-1332509348
that if you use "multi", prefer to call it "hideKey".

Reply:
Same for the other "hideKeys" instance.

Reply:
Have changed both the description and the parameter name to "HideKey"